Userland/Applications/CertificateSettings/CertificateStore.cpp

```diff
@@ -29,16 +29,7 @@ void CertificateStoreProxyModel::sort(int column, GUI::SortOrder sort_order)
 
 ErrorOr<void> CertificateStoreModel::load()
 {
-    auto cacert_file = TRY(Core::File::open("/etc/cacert.pem"sv, Core::File::OpenMode::Read));
-    auto data = TRY(cacert_file->read_until_eof());
-
-    auto user_cert_path = TRY(String::formatted("{}/.config/certs.pem", Core::StandardPaths::home_directory()));
-    if (FileSystem::exists(user_cert_path)) {
-        auto user_cert_file = TRY(Core::File::open(user_cert_path, Core::File::OpenMode::Read));
-        TRY(data.try_append(TRY(user_cert_file->read_until_eof())));
-    }
-
-    m_certificates = TRY(DefaultRootCACertificates::the().reload_certificates(data));
+    m_certificates = TRY(DefaultRootCACertificates::load_certificates());
```

Comment:
In the old design, we were updating the CA certificates in the singleton. You're now bypassing that, leaving the singleton with an old set of certificates. Is that intentional?

Reply:
The Singleton still get's repolutated every time a RequestServer starts. In the cert store it is not used at all, just the parsing.

Reply:
Does it still make sense to have it as a singleton then? Afaics, most of the usage is just loading the CA certificates once.

Reply:
That depends: If someone would change the Vector the Singleton holds from the outside? Would it change?

The scenario that might be interessting is not the opening of the cert store, but actually the importing of a new one. Could I inject the newly imported cert directly in other running processes that hold a reference to the Singleton if I change the Vector in the Singleton?

Reply:
> Could I inject the newly imported cert directly in other running processes that hold a reference to the Singleton if I change the Vector in the Singleton?

No, this is purely a singleton within a process, we don't cross process borders in userland except with services that utilize IPC to communicate.

So since your de-singletonizing some methods for CA certificates, it might no longer make sense to utilize a singleton pattern for the other use cases - or we could keep the singleton pattern as-is.

Reply:
Someone with more understanding of RequestServer and WebSocket would have to check that. In the test and the cert store, there is no use for the singleton.

Reply:
I was confused by the fact that the old `reload_certificates` did not actually update the object's state. Consider this resolved!